gas_tracker.go

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 Avi Misra
+ *
+ * Use of this work is governed by a MIT License.
+ * You may find a license copy in project root.
+ */
+
+package etherscan
+
+// GasEstiamte gets estiamted confirmation time (in seconds) at the given gas price
+func (c *Client) GasEstimate(gasPrice int) (confirmationTimeInSec string, err error) {
```

Comment:
Maybe we should make `confirmationTimeInSec` a `int`? An user of this method may expect it handy.

Reply:
An `int` is good, a `time.Duration` is better IMO.

Reply:
Cool, let's use time.Duration.

Reply:
See [384329f](https://github.com/nanmu42/etherscan-api/pull/74/commits/384329f2261b7e66a1ab6dc175af1010a537a8e1)